src/event-detail/ui/event-detail.html

```diff
@@ -101,7 +101,8 @@ <h1 class="title" ng-bind="::event.name"></h1>
                     <span translate-once="preview.labels_success" translate-values="{ addedLabel: '{{addedLabel}}' }"></span>
                   </p>
                   <p ng-if="labelResponse === 'unlabelError'" class="alert alert-danger">
-                    <span ng-bind="labelsError"></span>
+                    <span ng-if="labelsError !== 'Label not allowed'" ng-bind="labelsError"></span>
+                    <span ng-if="labelsError === 'Label not allowed'" translate-once="errors.labelNotAllowed"></span>
```

Comment:
I'm rusty on my Angular but is there perhaps a directive for translate + fallback? Then we could try translation and fall back to the raw string if no custom error message is specified, would make it easier to translate more messages afterwards perhaps

Reply:
My Angular is rusty as well 😅 , I didn't find something immediately.